LayoutTests/resources/ui-helper.js

```diff
@@ -1255,6 +1255,22 @@ window.UIHelper = class UIHelper {
         return new Promise(resolve => testRunner.runUIScript(uiScript, resolve));
     }
 
+    static async smartMagnifyAt(x, y)
+    {
+        if (!this.isWebKit2() || !this.isMac()) {
+            console.log('Smart magnify is only supported on macOS');
```

Comment:
Err, not true as a feature. Maybe true for this function? Can this be worded more like "we haven't implemented it in the test runner?"

Reply:
On iOS I think you could trigger it with the existing touch event synthesis. I suppose this code could generate the appropriate touch events there.